design/TF.md

```diff
@@ -1,79 +1,72 @@
+
+In order to ensure that control is unaffected by localization failure, we require that all preceding calculations do not use the map->odom transform. However, since trajectory following comes after planning in Autoware, it is almost impossible to prevent map->odom from being involved in trajectory calculations. Although this might be possible if Autoware planned like a human (who only uses route graph information from the map and can obtain geometry information from perception), it is very unlikely that an autonomous driving stack is capable of ensuring safety without using geometry information. 
 
-Therefore, regardless of any frame that control is done, it will still have effects of localization results. To make control not to jerk or do sudden steering at localization failure, we set the following requirements to Localization module:
-* localizatoin result must be continuous
-* localization must detect localization failure and should not use the result to update vehicle pose.
+Therefore, regardless of the frame in which it is done, control will still be affected by localization. To control a vehicle without jerking or sudden steering in the event of localization failure, we set the following requirements for the Localization module:
+* Localization results must be continuous
+* Localization failures must be detected and the vehicle's pose should not be updated with any failed localization results
 
-Additionally, Localization architecture assumes sequential Bayesian Filter, such as EKF and particle filter, to be used to integrate twist and pose. Since it can also integrate odometry information, it can update TF at high frequency. 
-As a result, all the merits of odom->base_link stated in REP-105 can be satisfied by map->base_link, and thus there is no need of setting odom frame. 
+Additionally, the Localization architecture assumes that twist and pose will be integrated with a sequential Bayesian Filter such as [EKF](https://en.wikipedia.org/wiki/Extended_Kalman_filter) or a [particle filter](https://en.wikipedia.org/wiki/Particle_filter)). Since it can also integrate odometry information, it can update TF at high frequency. As a result, all of the merits of odom->base_link stated in REP-105 can be satisfied by map->base_link, and thus there is no need to set the odom frame. 
```

Comment:
> Since it can also integrate odometry information, it can update TF at high frequency

In this sentence, what is 'it" referring to? A sequential Bayesian filter, the Localization module, or both?